### ISSUE TITLE
Add fitting constraints

### ISSUE THREAD

**Issue by leek2** [LLNL/Curvallis#16]
When doing a fit, sometimes the coefficients identified become completely ridiculous.  Is there some way to add constraints to the fitting algorithm so it doesn't go crazy?

**Comment by leek2** [comment]
Yes, scipy 17.0 added bounds.  We should check the version before taking advantage of it though, as some machines are still using older versions.

**Comment by WilliamIsMe1** [comment]
This is a duplication of issue #9. Also what counts as ridiculous for certain coefficients?
